packages/NGSadd.taxonomy/meta/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <Package>
-	<DisplayName>Taxonomy classification (size Gb)</DisplayName>
+	<DisplayName>Taxonomy classification (3.1 Gb)</DisplayName>
```

Comment:
As I know size info shows automatically. And we doesn't need to add this info manually.
![image](https://user-images.githubusercontent.com/5506989/33865643-345fa704-df25-11e7-8bc4-a98c84d31b50.png)

Reply:
Addressed in be511cd,
Thanks for pointing this out